### PR TITLE
add "-O1 -disable-llvm-optzns"

### PR DESCRIPTION
**Pull request by PeimingLiu**
This will keep the IR unoptimized but at the same time all the metadata for optimization will be kept

### CODE REVIEW

File: tests/data/integration/dataracebench/ll/logs/log.current [hunk 1]
```diff
@@ -0,0 +1,141 @@
+17:43:50 [debug] [racedetect] [Log.cpp:46] Logging configs. level=1, logFolder=logs, logFile=log.current, enableTerminal=false, enableFile=true
```
Comment (BradSwain) [inline tests/data/integration/dataracebench/ll/logs/log.current:1]: This file should be deleted

**Review by BradSwain — CHANGES_REQUESTED** [review]
The following files should be removed:
- tests/data/integration/dataracebench/ll/logs/log.current
- tests/data/integration/dataracebench/ll/modified.ll
- tests/data/integration/dataracebench/ll/o1.ll
- tests/data/integration/dataracebench/ll/output.txt

We should also figure out why the tests crash and update the Makefiles that build the other tests to include these flags as well.

**Comment by BradSwain** [comment]
The crash is caused by an unrecognized `__kmpc_*` call in DRB011.

The call is to a variant of `__kmpc_static_for_init` (`__kmpc_for_static_init_4u`).

This is already supported in  #99 so once that PR is merged the crash should go away.

File: tests/data/integration/dataracebench/ll/logs/log.current (hunk 1 above)
Comment (PeimingLiu) [reply tests/data/integration/dataracebench/ll/logs/log.current]: removed

**Comment by BradSwain** [comment]
I still need to add these flags to our other tests. I have mostly done it locally but have not had time to finish and push to this branch

**Comment by BradSwain** [comment]
I added a top level Makefile so that:
 - the same flags are used to build all test files
 - we can check the right compiler version is used
 - only need to run make in one place to generate test data

As a downside/side effect all the .ll files are now located side  by side with the source files.

@funemy Can you check to make sure I didn't miss anything? If it looks good feel free to merge.

**Pull request merged**